{
  "file_data": [
    {
      "desc": {
        "cause": "Under Investigation",
        "cluster": false,
        "cust_a": {
          "masked": true,
          "val": 4
        },
        "etr": "2020-12-20T16:00:00-0400",
        "start": "2020-12-20T10:42:00-0400"
      },
      "geom": {
        "p": [
          "{fujGdh|jK"
        ]
      },
      "id": "2",
      "title": "Outage Information"
    }
  ],
  "file_title": "030322"
}
